persistence-cassandra/core/src/main/scala/com/lightbend/lagom/internal/persistence/cassandra/CassandraOffsetStore.scala

```diff
@@ -54,9 +59,8 @@ private[lagom] abstract class CassandraOffsetStore(
   }
 
   protected def doPrepare(eventProcessorId: String, tag: String): Future[(Offset, PreparedStatement)] = {
-    implicit val timeout = Timeout(config.globalPrepareTimeout)
     for {
-      _ <- startupTask.askExecute()
+      _ <- startupTask
```

Comment:
It should be I guess + I followed code consistency with com.lightbend.lagom.internal.persistence.jdbc.SlickOffsetStore